pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -135,41 +136,62 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 		},
 	}
 
-	log.Debug("Creating CiliumNetworkPolicy/v2 CustomResourceDefinition...")
-	clusterCRD, err := clientset.ApiextensionsV1beta1().CustomResourceDefinitions().Create(res)
-	if err != nil && !errors.IsAlreadyExists(err) {
+	return createUpdateCRD(clientset, "CiliumNetworkPolicy/v2", res)
+}
+
+// createUpdateCRD ensures the CRD object is installed into the k8s cluster. It
+// will create or update the CRD and it's validation when needed
```

Comment:
it's --> its

Reply:
indeed it's. I'll make another PR to fixup typos and comments I owe you :)

Reply:
I got overzealous with my janitorial duties and merged this without realizing that I had just left a review comment, sorry about that :(

Reply:
Oh, it was you! I'm so used to tgraf merging things I assumed it was him. No worries!